### PR TITLE
"Add MERRA2 to ClimateFiller and Update Downloading using Backend GEE"

### PR DESCRIPTION
**Pull request by souidiyassine**

### CODE REVIEW

**Review by elhachimi-ch — CHANGES_REQUESTED** [review]
Delete the JSON file because the user needs to set up his own GEE credentials

File: climatefiller_credentials.json
Comment (elhachimi-ch) [inline climatefiller_credentials.json:1]: To be deleted

File: test-utils.ipynb
Comment (elhachimi-ch) [inline test-utils.ipynb:1]: Looks good!

File: utils.py
Comment (elhachimi-ch) [inline utils.py:1]: Good job!

**Review by elhachimi-ch — CHANGES_REQUESTED** [review]

File: climatefiller_credentials.json
Comment (souidiyassine) [reply climatefiller_credentials.json:1]: The requested change has been completed

**Review by elhachimi-ch — APPROVED** [review]

**Pull request closed without merging**